AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -278,6 +278,7 @@ static const ap_message STREAM_EXTENDED_STATUS_msgs[] = {
     MSG_NAV_CONTROLLER_OUTPUT,
     MSG_GPS_RAW,
     MSG_GPS_RTK,
+    MSG_GPS_STATUS,
```

Comment:
We can't stream this by default - *way* too much data for a normal telemetry link.

```suggestion
```

There are many ways a user could request this information from the autopilot as required - for example, a hypothetical satellites view in MissionPlanner could poke the autopilot to start sending it the data (and the autopilot would then make dynamic allocations to start collecting the data from the GPS device.